Sources/SwiftInspectorCore/Tests/TypeConformanceAnalyzerSpec.swift

```diff
@@ -40,26 +40,28 @@ final class TypeConformanceAnalyzerSpec: QuickSpec {
     }
 
     describe("analyze(fileURL:)") {
+      var result: TypeConformance!
```

Comment:
do you need the force unwrap? I looks like you use `?` when inspecting this value below

Reply:
Yup! Good idea